doc/api/cluster.md

```diff
@@ -336,7 +336,7 @@ added: v6.0.0
 
 * {boolean}
 
-Set by calling `.kill()` or `.disconnect()`. Until then, it is `undefined`.
+Set by calling `.kill()` or `.disconnect()`. Until then, it is `undefined` or `false`.
```

Comment:
Can you add an explanation as to when it is `undefined` and when it is `false`?

Reply:
Yeah sure